interoperability_report.py

```diff
@@ -326,8 +330,8 @@ def run_publisher_shape_main(
                         for x in range(0, MAX_SAMPLES_SAVED, 1):
                             # We select the numbers that identify the samples
                             # and we add them to samples_sent.
-                            pub_string = re.search('[0-9]{3} [0-9]{3}',
-                                    child_pub.before)
+                            pub_string = re.search('[0-9]{3} [0-9]{3} \[[0-9][0-9]\]',
```

Comment:
cannot we use [0-9]{2} ? we should also add a comment in the function that checks the test is correct, that only size < 99 is allowed to work correctly